packages/medusa/src/api/routes/admin/orders/get-order.ts

```diff
@@ -8,6 +9,11 @@ import { OrderService } from "../../../../services"
  * x-authenticated: true
  * parameters:
  *   - (path) id=* {string} The ID of the Order.
+ *   - (query) expand {string} Comma separated list of relations to include in the results.
+ *   - (query) fields {string} Comma separated list of fields to include in the results.
+ * x-codegen:
+ *   method: retrieve
+ *   queryParams: AdminGetOrdersOrderParams
```

Comment:
Created alias for FindParams
https://github.com/medusajs/medusa/blob/75f9c7a46c89bfa86b65a57d0ac0c7b79d6a58f1/packages/medusa-js/src/resources/admin/orders.ts#L34-L38